evalai/download_evalai_starters_kit.py

```diff
@@ -0,0 +1,13 @@
+import click
+import subprocess
+
+from evalai.utils.config import STARTERS_URL
+
+
+@click.group(invoke_without_command=True)
+def download_evalai_starters_kit():
+    """
+    Download the EvalAI-Starters kit.
+    """
+    download_path = click.prompt("Enter directory to store EvalAI-Starters in", type=str, hide_input=False)
+    subprocess.run(["git", "clone", STARTERS_URL, download_path])
```

Comment:
From what I am getting from this statement, you are trying to use `git` command line to clone the `evalai-starters` repository.  But, what if the user is not having `git` cli installed? This is kind of extra dependency from users side.

How about download repository codebase zip and unzip it to `download_path`?
The link can be:
```
http://github.com/cloud-cv/evalai-starters/zipball/master/
```
Refer from: https://stackoverflow.com/questions/2751227/how-to-download-source-in-zip-format-from-github

Reply:
I dispatched git from dependency. Does the code look good?

Reply:
Yes !